src/InvertedIndex.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * InvertedIndex class
+ */
+  class InvertedIndex {
+ /**
+  * constructor - contain class properties
+  */
+    constructor() {
+      this.allIndices = {};
+      this.fileContent = '';
+    }
+  /**
+    * createIndex - creates index for json files e.g {of:[0,1,2,3,4]}
```

Comment:
the example part might not be necessary. Your README might as well explain all this